tools/postgres/schema.sql

```diff
@@ -172,6 +175,10 @@ CREATE TABLE webknossos.dataset_mags(
   path TEXT,
   realPath TEXT,
   hasLocalData BOOLEAN NOT NULL DEFAULT FALSE,
+  axisOrder TEXT CONSTRAINT axisOrder_format CHECK (axisOrder ~ '^[xyzc]:[0-9]+(,[xyzc]:[0-9]+)+$'),
```

Comment:
TBH I’m not a huge fan of the custom axisOrder literal. How about using a jsonb column?